commands/summary.go

```diff
@@ -121,11 +121,20 @@ func addClientData(ctx context.Context, zipW *zip.Writer) {
 		logrus.Errorf("%s", err)
 		return
 	}
+	now := time.Now()
+	for _, agent := range status.AgentsInfo {
+		b, err := json.MarshalIndent(agent.Logs, "", "  ")
+		if err != nil {
+			logrus.Debugf("%s", err)
+			b = []byte(err.Error())
+		}
+		b = append(b, '\n')
+		addData(zipW, "logs/"+pointer.GetString(agent.AgentType)+".json", now, bytes.NewReader(b))
```

Comment:
what is the reason to make it json?
I think it would be much easier to read if it would be just a text with some separator.
somethink like
```
agent id: /agent_id/xxxx1
agent type: mongodb_exporter
lorem ipsum log texts begin
...
lorem ipsum log texts
===============================================
agent id: /agent_id/xxxx2
agent type: mysqld_exporter
lorem ipsum log texts begin
...
lorem ipsum log texts
```